modules/apps/segments/segments-test/src/testIntegration/java/com/liferay/segments/web/internal/display/context/test/SegmentsExperienceSelectorDisplayContextTest.java

```diff
@@ -170,11 +214,25 @@ private MockHttpServletRequest _getMockHttpServletRequest()
 			WebKeys.CURRENT_URL, "http://www.liferay.com");
 		mockHttpServletRequest.setAttribute(
 			WebKeys.THEME_DISPLAY, _getThemeDisplay());
+		mockHttpServletRequest.addParameter(
```

Comment:
As you only use it in a method, better if you set the parameter directly in the test you use it.